packages/grafana-data/src/index.ts

```diff
@@ -490,6 +490,8 @@ export type {
   ExploreLogsPanelState,
   SplitOpenOptions,
   SplitOpen,
+  TraceSearchProps,
```

Comment:
If this is truly internal to Grafana core you should export it in https://github.com/grafana/grafana/blob/main/packages/grafana-data/src/internal/index.ts instead. Adding the `@internal` comment doesn't prevent this from leaking out to the public realm.